craps/table/bet.py

```diff
@@ -1,133 +1,72 @@
 """
```

Comment:
This file is still pretty large. Is it worth moving some of the classes our and creating a package for them?

Reply:
I split the file into two separate files, one for concrete bets one for the abstracts and decorator